hosted/json-schemas/definitions.json

```diff
@@ -5,11 +5,11 @@
   "definitions": {
     "did": {
       "type": "string",
-      "pattern": "^did:([a-z0-9]+):((?:(?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))*:)*((?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))+))((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(\/[^#?]*)?([?][^#]*)?(#.*)?$"
```

Comment:
unfortunately `jq` turns this pattern into an invalid regex

Reply:
@jiyoontbd so i think as this PR currently stands, this current change _might_ break if a `did` has a path in it. how do we test these? i didn't see any test vectors with a _path_ so i wrote my own mini python test harness (hence the screenshotted error)

what do you recommend we do?

Reply:
come to think of it, the `did` _definition_ probably ought to live under [the web5 repo](https://github.com/TBD54566975/web5-spec/), and within that repo we ought to have test vectors for DIDs like this. think we could get there in this PR? or we can add an issue and follow-up subsequently

Reply:
fixed a typo (ty @jiyoontbd) and added the escaped slash [123217d](https://github.com/TBD54566975/tbdex/pull/364/commits/123217dfe0a33f1d07732251f72289ddc450be9f). suppose we'll keep this (corrected) for now as the status quo.

@jiyoontbd chatted about this offline wrt to moving it to web5 so i think maybe it's a bigger discussion between the protocol and the SDK teams, but i'll let y'all figure out the best path forward here!

Reply:
hmm, if we moved this to web5-spec, wouldn't each tbdex sdk now have to pull in that spec as a submodule just to be able to check if a DID is valid?

Reply:
JSON schema supports remote references (`"$ref": "{url}"`) for definitions, so we would add the definition in web5-spec, deploy it and add the reference here. But it doesn't need to be a blocker on this PR, but at the least we should open an Issue to track it b/c I believe that's the ideal architectural solution.

Reply:
https://github.com/TBD54566975/tbdex/issues/371